pkg/consensus/payload.go

```diff
@@ -79,28 +89,50 @@ func (p *Payload) SetPayload(pl interface{}) {
 }
 
 // GetChangeView implements payload.ConsensusPayload interface.
-func (p Payload) GetChangeView() payload.ChangeView { return p.payload.(payload.ChangeView) }
+func (p Payload) GetChangeView() payload.ChangeView {
+	if p.message == nil {
```

Comment:
Do we need this if in every method? Is it possible to parse `message` once it reaches consensus service on CN node?